features/sp-solution-features/org.wso2.carbon.sp.solutions.ei.analytics.feature/pom.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>sp-solution-features</artifactId>
+        <groupId>org.wso2.analytics.solutions</groupId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <name>WSO2 Stream Processor - EI Analytics Solution Feature</name>
+    <artifactId>org.wso2.carbon.sp.solutions.ei.analytics.feature</artifactId>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.wso2.analytics.solutions</groupId>
+            <artifactId>wso2sp-solutions.ei-analytics.statschart</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
```

Comment:
remove versions here and in other dependencies as well.

Reply:
Fixed in 53951238f84729f0c325006f8def576a3c702eda